pulpcore/tests/functional/__init__.py

```diff
@@ -106,6 +160,11 @@ def pytest_configure(config):
 # API Clients
 
 
+@pytest.fixture(scope="session")
+def pulp_cfg(bindings_cfg):
```

Comment:
```suggestion
def pulp_cfg():
```
Also i thought we could just use `django.config.settings` and it would do the dynaconf loading. Can you elaborate why this may not be possible?

Is that not what we need? https://github.com/pulp/pulpcore/blob/main/pulpcore/tests/functional/__init__.py#L473

Reply:
Ah 😮. I have already forgotten this fixture. I can simplify some workflows (e.g., in pulp_container).

The idea was to migrate the `pulp_cfg` fixture to pulpcore. Only `settings.CONTENT_ORIGIN`  can be used as a replacement for `pulp_cfg.get_base_url()`.

For settings like `aiohttp_fixtures_origin` or `fixtures_url`, we still need to access the configuration file defined initially for pulp_smash. In oci_env, there is `fixtures_url="http://pulp-fixtures:8080/`, but in a vagrant box, there is `fixtures_url=https://fixtures.pulpproject.org/`. I think we should still read these two settings from the configuration file since not all of us run fixtures locally. What do you think? Should I rather open PRs against the oci_env and pulp_installer repositories where I will set environment variables for these settings and then hard-code default values for them as we did in `get_bindings_config`?

Reply:
Oh, OK, I was mislead by seeing dynaconf. But this one is reading the smash config. Is fixtures_url really the only one thing we need from there? Then yes, i'd prefer not adding a fixture we will have a hard time removing over the course of time.

Reply:
We deal with two settings here:
- aiohttp_fixtures_origin -> that is the origin for a server used by pulp_file (e.g., the `gen_bad_response_fixture_server` fixture); this can be hard-coded because it looks like we always set it to "127.0.0.1".
- fixtures_url -> is the origin for the fixtures we pull content from; could be set to a remote or local URL; this should be read from the actual settings from now

I will try to project this to the PR once again.

Reply:
We said, our tests always run on localhost (one of the make-it-simple design goals). So let's use this fact and set `aiohttp_fixtures_origin` hardcoded (should we remove it; turn it into a constant?).

For `fixtures_url` you suggested reading it from Environment variables with a decent default. I would like to do that. Also keep in mind that another goal is to not use the `fixtures_url` at all eventually (this should allow for _some_ amount of duct-tape here). Can you add comments for this plan so we don't forget?

Reply:
An afterthought: To me "getting away from pulp-smash" also includes "stopping to use smash config".

Reply:
Okay. It makes sense. I just misunderstood your choice of words. I am going to declare a new environment variable for `fixtures_url` (in oci_env and pulp_installer as well).